changelogs/fragments/411-lint-collection-docs.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - "Add ``lint-collection-docs`` subcommand to ``antsibull-docs``. It behaves identical to ``antsibull-lint collection-docs`` (https://github.com/ansible-community/antsibull/pull/411)."
```

Comment:
(optional) link to discussion around splitting out lint into subcommands?

Reply:
Done.